wagon.py

```diff
@@ -683,7 +688,8 @@ def create(source,
            validate_archive=False,
            wheel_args='',
            archive_format='zip',
-           build_tag=''):
+           build_tag='',
+           pip_paths=[None]):
```

Comment:
@AdarShaked cannot we just pass `None` by default here to the `pip_paths` because list is mutable object and the param here should immutable